great_expectations/datasource/data_connector/inferred_asset_sql_data_connector.py

```diff
@@ -166,13 +166,13 @@ def _refresh_introspected_assets_cache(
                 "table_name": metadata["table_name"],
                 "type": metadata["type"],
             }
-            if not splitter_method is None:
+            if splitter_method is not None:
                 data_asset_config["splitter_method"] = splitter_method
-            if not splitter_kwargs is None:
+            if splitter_kwargs is not None:
                 data_asset_config["splitter_kwargs"] = splitter_kwargs
-            if not sampling_method is None:
+            if sampling_method is not None:
                 data_asset_config["sampling_method"] = sampling_method
-            if not sampling_kwargs is None:
+            if sampling_kwargs is not None:
```

Comment:
Here's another case of actual run-time changes.
As suggested by flake8

https://www.flake8rules.com/rules/E714.html

Reply:
love it!